controllers/cloud.redhat.com/helpers/namespaces.go

```diff
@@ -19,66 +19,70 @@ import (
 )
 
 func CreateNamespace(ctx context.Context, cl client.Client, pool *crd.NamespacePool) (string, error) {
-	namespace := core.Namespace{}
+	ns := core.Namespace{}
 
-	namespace.Name = fmt.Sprintf("ephemeral-%s", strings.ToLower(utils.RandString(6)))
+	ns.Name = fmt.Sprintf("ephemeral-%s", strings.ToLower(utils.RandString(6)))
 
 	if pool.Spec.Local {
-		if err := cl.Create(ctx, &namespace); err != nil {
-			return "", fmt.Errorf("could not create namespace [%s]: %w", namespace.Name, err)
+		if err := cl.Create(ctx, &ns); err != nil {
+			return ns.Name, fmt.Errorf("could not create namespace [%s]: %w", ns.Name, err)
 		}
 	} else {
 		project := projectv1.ProjectRequest{}
-		project.Name = namespace.Name
+		project.Name = ns.Name
 		if err := cl.Create(ctx, &project); err != nil {
-			return "", fmt.Errorf("could not create project [%s]: %w", project.Name, err)
+			return ns.Name, fmt.Errorf("could not create project [%s]: %w", project.Name, err)
 		}
 	}
 
+	return ns.Name, nil
+}
+
+func UpdateNamespaceResources(ctx context.Context, cl client.Client, pool *crd.NamespacePool, nsName string) (core.Namespace, error) {
 	// WORKAROUND: Can't set annotations and ownerref on project request during create
```

Comment:
Remove this comment because its no longer a work around. We're just getting the namespace now!